x/cscalist/keeper/proposal.go

```diff
@@ -0,0 +1,50 @@
+package keeper
+
+import (
+	"cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	"github.com/rarimo/rarimo-core/x/cscalist/types"
+)
+
+func (k Keeper) EditCSCAListProposal(ctx sdk.Context, proposal *types.EditCSCAListProposal) error {
+	tree := Treap{k}
+	root := k.GetRootKey(ctx)
+
+	for _, leaf := range proposal.GetToRemove() {
```

Comment:
why additional stack call? proposal.GetToRemove()
just use proposal.ToRemove and access the field directly if this function (i believe) does not contain any additional logic

Reply:
+, we have validated it beforehand